[2023-11-15T09:00:06.763+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_projet.create_table_urgences manual__2023-11-15T09:00:04.521111+00:00 [queued]>
[2023-11-15T09:00:06.768+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_projet.create_table_urgences manual__2023-11-15T09:00:04.521111+00:00 [queued]>
[2023-11-15T09:00:06.769+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-15T09:00:06.775+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_table_urgences> on 2023-11-15 09:00:04.521111+00:00
[2023-11-15T09:00:06.779+0000] {standard_task_runner.py:57} INFO - Started process 1240 to run task
[2023-11-15T09:00:06.781+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_projet', 'create_table_urgences', 'manual__2023-11-15T09:00:04.521111+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/ETL_projet.py', '--cfg-path', '/tmp/tmpb2x7bmhb']
[2023-11-15T09:00:06.781+0000] {standard_task_runner.py:85} INFO - Job 64: Subtask create_table_urgences
[2023-11-15T09:00:06.801+0000] {task_command.py:416} INFO - Running <TaskInstance: ETL_projet.create_table_urgences manual__2023-11-15T09:00:04.521111+00:00 [running]> on host 48045a1653b9
[2023-11-15T09:00:06.829+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='a***' AIRFLOW_CTX_DAG_ID='ETL_projet' AIRFLOW_CTX_TASK_ID='create_table_urgences' AIRFLOW_CTX_EXECUTION_DATE='2023-11-15T09:00:04.521111+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-15T09:00:04.521111+00:00'
[2023-11-15T09:00:06.830+0000] {sql.py:274} INFO - Executing: CREATE TABLE IF NOT EXISTS urgences_covid (
    id serial PRIMARY KEY,
    dep INT REFERENCES departments_region(num_dep),
    date_de_passage DATE,
    sursaud_cl_age_corona INT REFERENCES tranche_age(Code),
    nbre_pass_corona INT,
    nbre_pass_tot INT,
    nbre_hospit_corona INT,
    nbre_pass_corona_h INT,
    nbre_pass_corona_f INT,
    nbre_pass_tot_h INT,
    nbre_pass_tot_f INT,
    nbre_hospit_corona_h INT,
    nbre_hospit_corona_f INT
);
[2023-11-15T09:00:06.833+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-15T09:00:06.836+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-15T09:00:06.840+0000] {sql.py:418} INFO - Running statement: CREATE TABLE IF NOT EXISTS urgences_covid (
    id serial PRIMARY KEY,
    dep INT REFERENCES departments_region(num_dep),
    date_de_passage DATE,
    sursaud_cl_age_corona INT REFERENCES tranche_age(Code),
    nbre_pass_corona INT,
    nbre_pass_tot INT,
    nbre_hospit_corona INT,
    nbre_pass_corona_h INT,
    nbre_pass_corona_f INT,
    nbre_pass_tot_h INT,
    nbre_pass_tot_f INT,
    nbre_hospit_corona_h INT,
    nbre_hospit_corona_f INT
);, parameters: None
[2023-11-15T09:00:06.850+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_projet, task_id=create_table_urgences, execution_date=20231115T090004, start_date=20231115T090006, end_date=20231115T090006
[2023-11-15T09:00:06.874+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-15T09:00:06.884+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
